src/pages/Login/Login.scss

```
@@ -0,0 +1,136 @@
+@import '../../styles/mixin.scss';
+@import '../../styles/variables.scss';
+
+@mixin sectionHeaderSpan {
+  margin-bottom: 20px;
+  font-weight: bolder;
+}
+
+@mixin loginButton {
+  width: 100%;
+  padding: 10px 0;
+  border: 1px solid black;
+  background-color: white;
+  transition: all 0.2s ease-in-out;
+  cursor: pointer;
+}
+
+@mixin loginButtonHover {
+  color: white;
+  background-color: black;
+}
+
+.display {
+  opacity: 0;
+}
+
+.login {
+  @include flexContainer(center, center);
+  background-color: rgba(0, 0, 0, 0.2);
+  font-size: 14px;
+  font-family: Arial, Helvetica, sans-serif;
+
+  .container {
+    @include flexContainer(flex-start, center);
+    padding: 20px;
+    background-color: white;
+    border: 1px solid black;
+
+    .section-left {
+      width: 50%;
+      height: 100%;
+
+      .login-header {
+        span {
+          display: block;
+          margin-bottom: 10px;
+        }
+        .left-section-header {
+          @include sectionHeaderSpan;
+        }
+      }
+
+      .login-main {
+        * {
+          display: block;
+        }
+        .forget-password {
+          margin-top: 20px;
+          color: rgba(0, 0, 0, 0.4);
+        }
```

```suggestion
      .login-main {
        .forget-password {
          display: block;
          margin-top: 20px;
          color: rgba(0, 0, 0, 0.4);
        }
```
직접 구현해 본 결과 이와같이 구현해도 UI 적으로 큰 차이를 보이지 않습니다.

---

감사합니다 수정완료입니다 !